cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/handlers/NGSIRestHandler.java

```diff
@@ -238,6 +243,9 @@ public void configure(Context context) {
                     
                     servicePath = headerValue;
                     
+                    break;
+                case CommonConstants.HEADER_NGSI_VERSION:
```

Comment:
Is this a header????

Reply:
Yes, header `ngsiv2-attrsformat` indicates if the request is processed as NGSIv2 or NGSIv1.

Reply:
That header is described in [the NGSIv2 specification](http://telefonicaid.github.io/fiware-orion/api/v2/stable) section "Notification Messages":

> Notifications must include the `Ngsiv2-AttrsFormat` HTTP header with the value of the format of the associated subscription, so that notification receivers are aware of the format without needing to process the notification payload.

Reply:
Note that this strategy (i.e. use the same endpoint for NGSIv1 and NGSIv2 notification and distingish based in `ngsiv2-attrsformat`) is also the one used by Perseo CEP.